core/chapters/c06_lists.py

```diff
@@ -419,9 +419,140 @@ class index_error(Step):
         def check(self):
             return "IndexError" in self.result
 
-    class introducing_len_and_range(VerbatimStep):
+    class improve_with_list_and_loop(VerbatimStep):
         """
 There you go. `words[4]` and beyond don't exist, so trying that will give you an error.
+That first program is a bit repetitive. Let's improve it with a list and a loop!
+
+__program_indented__
+        """
+
+        def program(self):
```

Comment:
```suggestion
        auto_translate_program = False

        def program(self):
```

That's a new part of the API. `['This', 'is', 'a', 'list']` isn't easily translated automatically, so the program needs to be translated manually.

Reply:
OK, didn't know that!